### DISCUSSION TITLE
allow support for multiple hit error bars (and UR counters)

### DISCUSSION THREAD

**Discussion by HaloExpert422** [ppy/osu#21514] [Ideas]
hi. i'm a dual standard/taiko player who has a right hand with really garbage rhythmic ability.

i think the ability to have multiple hit error bars and unstable rate counters for different keys (e.g. different hit error bars for k1 and k2, or for different sides of a taiko drum) would be really useful for players like me, and i don't think it would be too difficult to implement

maybe just have an additional option for BarHitErrorMeter and UnstableRateCounter that would allow you to turn on and off polling of certain keys for each object, and hitting notes with keys that aren't those just doesn't register on them

![mheb_lazer](https://user-images.githubusercontent.com/20484272/205546503-678a91c2-1d52-42ef-b6ca-02eae6904083.png)

and i guess at that point you could also apply this to other things like the clicks per second counter if it's not hard to implement

**Comment by peppy** [comment]
Interesting idea!
